_drafts/2015-05-26-welcome-to-jekyll.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "On Seminars"
+date:   2015-05-26 12:03:10
```

Comment:
I don't think the date is required either - having it in the file name seems to be sufficient. 

The "front matter" for one of the previous posts looks like so: 

```
---
title: You're gonna need a smaller room
layout: post
author: Scott Forman
image: scott.png
---
```

If I knew what I was doing I could probably figure out the image based on the author, but for now I'm just specifying it explicitly.